### DISCUSSION TITLE
Black & White Theme

### DISCUSSION THREAD

**Discussion by jacksonblankenship** [chakra-ui/chakra-ui#4338] [Q&A] [answered]
First of all, hello! New to Chakra here. The branding for the application we're building is very simple - black and white. I've noticed that Chakra doesn't include "black" as a default color scheme, but we'd like to implement one and have that set as default. Does anybody know how much effort something like that would take? I tried to implement the Button component alone and it appeared to be more involved than anticipated.

**Comment by aaronkik** [comment, marked as answer]
Hi,

It doesn't take much effort at all, you just need to extend the theme and pick the pallete of colors you want to use. Then tell chakra to use that as the default theme. Just edit the color codes in the sandbox link and you will notice the changes.

Sandbox link: https://codesandbox.io/s/chakra-custom-theme-t0o2r

Documentation: https://chakra-ui.com/docs/theming/customize-theme

**Reply by jacksonblankenship** [reply]
The codesandbox link definitely makes it much easier to work with. Thanks!
